src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -141,75 +175,65 @@ type internal Let =
     }
 
 /// <summary>
-/// A <c>return</c> statement.
+/// An <c>update</c> statement.
 /// </summary>
-type internal Return =
+type internal UpdateStatement =
     {
         /// <summary>
-        /// The <c>return</c> keyword.
+        /// The <c>set</c> keyword.
         /// </summary>
-        ReturnKeyword: Terminal
+        SetKeyword: Terminal
 
-        /// The returned expression.
-        Expression: Expression
+        /// The identifier being updated.
+        Name: Terminal
+
+        /// The update operator.
```

Comment:
I feel it is worth calling out that the equal sign is included here:
```suggestion
        /// The update operator, e.g. <c>+=</c>.
```

Reply:
done